tutorials/last-value_cache/c++11/USER_QOS_PROFILES.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ (c) 2024 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+ RTI grants Licensee a license to use, modify, compile, and create derivative
+ works of the Software.  Licensee has the right to distribute object form only
+ for use with RTI products.  The Software is provided "as is", with no warranty
+ of any type, including any warranty for fitness for any purpose. RTI is under
+ no obligation to maintain or support the Software.  RTI shall not be liable for
+ any incidental or consequential damages arising out of the use or inability to
+ use the software.
+ -->
+<dds xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://community.rti.com/schema/current/rti_dds_profiles.xsd">
+    <qos_library name="MyLibrary">
+        <qos_profile name="MyProfile" is_default_qos="true">
+            <base_name>
+                <element>BuiltinQosLib::Generic.KeepLastReliable.TransientLocal</element>
```

Comment:
Should I comment this, use `<element>BuiltinQosLib::Generic.StrictReliable</element>` and add a comment that explains that if you don't want your data to be volatile you have to uncomment `<element>BuiltinQosLib::Generic.KeepLastReliable.TransientLocal</element>` and comment  `<element>BuiltinQosLib::Generic.StrictReliable</element>`? Or should we use  `<element>BuiltinQosLib::Generic.KeepLastReliable.TransientLocal</element>` out of the box for the example?